modules/services/karabiner-elements/default.nix

```diff
@@ -9,12 +9,18 @@ let
 in
 
 {
-  options = {
-    services.karabiner-elements.enable = mkEnableOption "Karabiner-Elements";
+  options.services.karabiner-elements = {
+    enable = mkEnableOption "Karabiner-Elements";
+    package = mkOption {
+      type = types.package;
+      default = pkgs.karabiner-elements;
+      defaultText = "pkgs.karabiner-elements";
+      description = "Karabiner-elements derivation to use.";
+    };
```

Comment:
```suggestion
    package = mkPackageOption pkgs "karabiner-elements" { };
```

Reply:
you're totally right, I updated the PR with the changes

(I didn't have my dev environment yet on this mac 👀)